boards/arm/b_u585i_iot02a/b_u585i_iot02a.dts

```diff
@@ -17,6 +17,10 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
+		zephyr,bt-uart=&uart4;
+		zephyr,bt-c2h-uart=&uart4;
```

Comment:
I would rather see that defined in the module part

Reply:
> I would rather see that defined in the module part

Not sure I understand this correctly. Because this configuration is related to the main microcontroller (U585i), and the UART4 is only connected to the BLE module via this link. If we put it in module device three, how we can address the main application how find the proper UART port for BLE communications?

Reply:
This comment applied to `zephyr,bt-c2h-uart`. Sorry, it was misleading.
```
   * - zephyr,bt-c2h-uart
     - Selects the UART used for host communication in the
       :ref:`bluetooth-hci-uart-sample`
```
This sample applies to the module, not the host processor.